exdb/static/exdb/js/edit_experience.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
```

Comment:
This file should be merged with `experience_approval.js` since they are the same file.